Makefile

```diff
@@ -151,3 +155,7 @@ node_modules/.uptodate: package.json
 .PHONY: python
 python:
 	@./bin/install-python
+
+.PHONY: gulp
+gulp: node_modules/.uptodate
+	$(GULP) $(args)
```

Comment:
This enables `make gulp args=<ARGS>` which I used as the gulp command format in some of the docs. You can still just install gulp yourself and use it instead of `make gulp` but I wanted to make that unnecessary